theme/src/components/drawer.js

```diff
@@ -4,16 +4,18 @@ import React from 'react'
 import {FocusOn} from 'react-focus-on'
 
 function Drawer({isOpen, onDismiss, children}) {
+  const linksBelowTheFoldBugEvents = {
+    onMouseDown: event => event.preventDefault(),
+    onClick: event => event.target.focus(),
+  }
   return (
     <AnimatePresence>
       {isOpen ? (
-        // eslint-disable-next-line jsx-a11y/click-events-have-key-events,jsx-a11y/no-static-element-interactions
         <div
           // These event handlers fix a bug that caused links below the fold
           // to be unclickable in macOS Safari.
           // Reference: https://github.com/theKashey/react-focus-lock/issues/79
-          onMouseDown={event => event.preventDefault()}
-          onClick={event => event.target.focus()}
```

Comment:
This seems to be one of the rules that are a bit too noisy from `jsx-a11y` and I think is an example of why the disable is valid (based on the comment) and we should keep it as-is along with using it as an example for why this rule shouldn't be reported in the scorecard.

Reply:
I would love to advocate for that. This rule `jsx-a11y/no-static-element-interactions` while preventing major issues, also hinders these kind of behind-the-scenes hacks.

Reply:
Yeah I'll put back the code for this change.